client/src/components/FeedPost/index.tsx

```diff
@@ -3,64 +3,143 @@ import '../../styles/style.scss';
 import '../Post/style.scss';
 import noAvatar from '../../assets/noAvatar.svg';
 import {Link} from 'react-router-dom';
+import TextareaAutosize from 'react-textarea-autosize';
+import { formatDescription } from '../../helpers/regex.description';
+import { IUserData } from '../Profile';
 
-export default class FeedPost extends React.Component<any> {
+interface IBody {
+    userId: string;
+    postId: string;
+}
+
+interface IProps {
+    id: any;
+    newsFeed: any;
+    author: any;
+    image: any;
+    description: string;
+    authorsOfLike: any;
+    user: IUserData;
+    getNewsFeedAsync: () => void;
+    getMoreNewsFeedAsync: (page: number) => void;
+    addLoggedUserLike: (loggedUserId: string, postId: string, authorsOfLike: []) => void;
+    removeLoggedUserLike: (loggedUserId: string, postId: string, authorsOfLike: []) => void;
+    addLike: (body: IBody) => void;
+    setCountOfLikes: (countOfLikes: number) => void;
+    deleteLike: (body: IBody) => void;
+    countOfLikes: number;
+    likeExist: boolean;
+    checkUserLikeExist: (doesExist: boolean) => void;
+    addNextFeedPosts: (pageNumber: number) => void;
+    loggedId: string;
+    loggedUsername: string;
```

Comment:
loggedId: string;
    loggedUsername: string;

чому б не зробити loggedUser={loggedId, loggedUsername} ?

Reply:
because in redax we have this variable not an object but two separate lines. ANd its logic for likes, this logic did Ostap and I only you something similar to my feed.